enterprise/coderd/provisionerkeys_test.go

```diff
@@ -133,3 +133,38 @@ func TestProvisionerKeys(t *testing.T) {
 	err = orgAdmin.DeleteProvisionerKey(ctx, owner.OrganizationID, codersdk.ProvisionerKeyNamePSK)
 	require.ErrorContains(t, err, "reserved")
 }
+
+func TestProvisionerKeyTags(t *testing.T) {
+	t.Parallel()
+	t.Run("GetTags", func(t *testing.T) {
+		t.Parallel()
+
+		ctx, cancel := context.WithTimeout(context.Background(), testutil.WaitLong*10)
```

Comment:
Why do you need to wait 250 seconds?

Also, there's `testutil.Context` which will save you a line of code here.

Reply:
I think this may be copy pasta from the above test.